core/document.adoc

```diff
@@ -0,0 +1,70 @@
+= OGC API - Processes - Part 1: Core
+:doctype: standard
+:docsubtype: Interface
```

Comment:
There is no such thing as an "Interface" subtype: https://www.metanorma.org/author/ogc/ref/document-attributes/#ogc-docsubtype

This should probably be an `implementation`.

Reply:
Done.